README.md

```diff
@@ -204,44 +204,64 @@ Prior to cloning the repository and attempting to build the project, ensure that
 To use test datasets you will need:
 - Git Large File Storage, version 3.0.2+
 
-### Building the project (first option: with tests)
+### Building the project
+#### Building the Python module using pip (this is what you probably want if you are not a developer)
+**NOTE**: this step is mandatory for setting up the Command Line Interface for Desbordante.
+
 Firstly, navigate to a desired directory.
```

Comment:
```suggestion
Firstly, navigate to your directory of choice.
```

Reply:
You're free to reject it, but "of choice" is more common. Search "to a directory of choice" and "to a desired directory" in Google and the numbers will be significantly different.

Reply:
OK, I agree, good point.